jobs/scripts/plip-py3.sh

```diff
@@ -1,22 +1,25 @@
 #!/bin/sh
-pip install -r requirements.txt
+pip install -Ur requirements.txt
 buildout buildout:git-clone-depth=1 -c py3.cfg
-cat bin/test
 
 return_code="all_right"
 
-# using the alltests recipe runs not enough tests (need to investigate)...
-# bin/alltests -vvv --xml || return_code=$?
+export PATH="/usr/lib/chromium-browser:$PATH"
+export ROBOT_BROWSER=chrome
+export ROBOTSUITE_PREFIX=ROBOT
 
-# running all tests breaks test-isolation. See https://github.com/plone/Products.CMFPlone/issues/2459 ...
-# ./bin/test --xml -vvv
+alias xvfb-wrap="xvfb-run -a --server-args='-screen 0 1920x1200x24'"
```

Comment:
I didn't bothered changing this one as @pbauer is still fiddling with it and whenever I update the jobs with ``jenkins-job-builder`` I always skip it, eventually when @pbauer says so, we should get it back to the regular jobs though, can we do that already @pbauer ?